test/greet-test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const greet = require ('../lib/greet.js');
+const expect = require ('chai').expect;
+
+describe('Greeting Module',function() {
+  describe('#sayHi', function (){
+    it('should return hello cayla!', function(){
+      var result = greet.sayHi('cayla');
+      expect(greet).to.have.property('sayHi');
+      expect(result).to.equal('hello cayla!');
+    });
+
+    it('should throw a missing name name error', function(){
+      var result = greet.sayHi;
```

Comment:
Great job storing the result of the method in a variable to more cleanly test below.